runtime/src/bank.rs

```diff
@@ -2833,35 +2806,6 @@ mod tests {
         assert_eq!(bank1.capitalization(), 42 * 42);
     }
 
-    #[test]
-    fn test_bank_inflation() {
```

Comment:
Is this ok to remove this? At first look, this doesn't like a storage-centric unit-test...

Reply:
This test relied on storage rewards which are now gone and the same intent is cases are covered later in this test: https://github.com/solana-labs/solana/blob/4e4a21f9b7086bc009feb14cbdffe39438f41498/runtime/src/bank.rs#L4125

Reply:
I see! thanks for explaining!